Supremicus/DawnTitleCard.py

```diff
@@ -0,0 +1,545 @@
+                name='Separator Character',
+                identifier='separator',
+                description='Character to separate season and episode text.',
+                tooltip='Default is <v>•</v>.'
+            ),
+            Extra(
+                name='Horizontal Alignment',
+                identifier='h_align',
+                description='Horizontal alignment of text.',
+                tooltip='Either <v>left</v>, <v>center</v> or <v>right</v>. Default is <v>left</v>.'
+            ),
+            Extra(
+                name='CRT TV Overlay',
+                identifier='crt_overlay',
+                description='Enable CRT TV overlay.',
+                tooltip=(
+                    'Enable CRT TV overlay: <v>nobezel</v> or <v>bezel</v>.<br>'
+                    'Default is <v>None</v>.'
+                ),
+            ),
+            Extra(
+                name='CRT TV Watched/Unwatched Overlay',
+                identifier='crt_state_overlay',
+                description='Enable CRT TV Watched/Unwatched overlay.',
+                tooltip=(
+                    'Enable CRT TV Watched/Unwatched overlay: <v>True</v> / <v>False</v>. Default is <v>False</v>.<br>'
+                    'Will only work if CRT TV Overlay is enabled.'
+                ),
+            ),
+            Extra(
+                name='Gradient Omission',
+                identifier='omit_gradient',
+                description='Whether to omit the gradient overlay',
+                tooltip=(
+                    'Either <v>True</v> or <v>False</v>. Set to <v>False</v> if you have '
+                    'trouble reading text on brighter images.<br>Default is <v>True</v>.'
+                ),
+            ),
+        ],
+        description=[
+            "Produce TitleCards with left, centered or right aligned text on the bottom.",
+            "Optional CRT TV overlay for shows like Stranger Things with a retro theme.",
+        ]
+    )
+
+    class CardModel(BaseCardTypeCustomFontAllText):
+        episode_text_vertical_shift: int = 0
+        title_text_horizontal_shift: int = 0
+        stroke_color: str = 'black'
+        separator: str = '•'
+        h_align: Literal['left', 'center', 'right'] = 'left'
+        symbol: str = None
+        crt_overlay: str = None
+        crt_state_overlay: bool = False
+        omit_gradient: bool = True
+
+    """Characteristics for title splitting by this class"""
+    TITLE_CHARACTERISTICS: SplitCharacteristics = {
+        'max_line_width': 28,
+        'max_line_count': 4,
+        'style': 'bottom',
+    }
+
+    """Characteristics of the default title font"""
+    TITLE_FONT = str(RemoteFile('Supremicus', 'ref/fonts/HelveticaNeue-Bold.ttf'))
+    TITLE_COLOR = 'white'
+    FONT_REPLACEMENTS = {}
+
+    """Whether this CardType uses season titles for archival purposes"""
+    USES_SEASON_TITLE = True
+
+    """Standard class has standard archive name"""
+    ARCHIVE_NAME = 'dawn'
```

Comment:
```suggestion
    ARCHIVE_NAME = 'Dawn'
```
This is just to better align with the archive naming convention of all other card types